pkg/api/types.go

```diff
@@ -311,3 +327,53 @@ func (p *PullPolicy) Set(v string) error {
 	}
 	return nil
 }
+
+// IsSafeFilename checks if the filename provided does not contain any invalid
+// characters
+func IsSafeFilename(name string) bool {
```

Comment:
what's the intent of this?
why blacklisting instead of whitelisting?
why is `•` there, but not any other funny unicode symbol?